test/functional/interface_bitcoin_cli.py

```diff
@@ -72,8 +73,48 @@ def run_test(self):
             assert_equal(cli_get_info['paytxfee'], wallet_info['paytxfee'])
             assert_equal(cli_get_info['relayfee'], network_info['relayfee'])
             assert_equal(self.nodes[0].cli.getwalletinfo(), wallet_info)
+
+            # Setup to test -getinfo and -rpcwallet= with multiple wallets.
+            wallets = ['', 'Encrypted', 'secret']
+            amounts = [Decimal('59.999928'), Decimal(9), Decimal(31)]
+            self.nodes[0].createwallet(wallet_name=wallets[1])
+            self.nodes[0].createwallet(wallet_name=wallets[2])
+            w1 = self.nodes[0].get_wallet_rpc(wallets[0])
+            w2 = self.nodes[0].get_wallet_rpc(wallets[1])
+            w3 = self.nodes[0].get_wallet_rpc(wallets[2])
+            w1.walletpassphrase(password, 60)
```

Comment:
Just as a paranoid check, could increasing the timeout here reduce the chance of a flaky test (I'm not familiar enough with the CI framework to answer this definitively)?

Reply:
I arbitrarily used the default `rpc_timeout`. But you're right to mention it; the magic value is kind of sucky and non-informative. Changed it to:
```diff
-            w1.walletpassphrase(password, 60)
+            w1.walletpassphrase(password, self.rpc_timeout)  # 60 sec by default
```